app/ui-angular/pom.xml

```
@@ -121,39 +124,47 @@
           </execution>
           <execution>
             <id>yarn ng lint</id>
+            <phase>compile</phase>
             <goals>
               <goal>yarn</goal>
             </goals>
             <configuration>
               <arguments>lint</arguments>
+              <yarnInheritsProxyConfigFromMaven>false</yarnInheritsProxyConfigFromMaven>
             </configuration>
           </execution>
           <execution>
             <id>yarn ng test</id>
+            <phase>compile</phase>
```

```suggestion
            <phase>test</phase>
```

Probably makes more sense than `compile`

---

Changed